plugins/modules/web_infrastructure/nomad_job.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, FERREIRA Christophe <christophe.ferreira@cnaf.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: nomad_job
+author: "FERREIRA Christophe (@chris93111)"
+version_added: "1.2.0"
+short_description: Launch an Nomad Job.
+description:
+    - Launch an Nomad job. See
+      U(https://www.nomadproject.io/api-docs/jobs/) for an overview.
+options:
+    host:
+      description:
+        - FQDN of nomad server.
+      required: true
+      type: str
+    secure:
+      description:
+        - TLS/SSL connection.
+      type: bool
+      default: false
+    timeout:
+      description:
+        - Timeout (in seconds) for the request to nomad.
+      type: int
+      default: 5
+    validate_certs:
+      description:
+        - Enable TLS/SSL certificate validation.
+      type: bool
+      default: true
+    cert:
+      description:
+        - Path of certificate TLS/SSL.
+      type: path
+    key:
+      description:
+        - Path of certificate key TLS/SSL.
+      type: path
+    namespace:
+      description:
+        - Namespace for nomad.
+      type: str
+    token:
+      description:
+        - ACL token for authentification.
+      type: str
+    name:
+      description:
+        - Name of job for delete,stop and start job without source.
+      type: str
+    state:
+      description:
+        - Deploy or remove job
+      choices: ["present", "absent"]
+      required: True
+      type: str
+    force_start:
+      description:
+        - Force job to started
+      type: bool
+      default: false
+    source_json:
+      description:
+        - Source of json nomad job
+      type: json
+    source_hcl:
+      description:
+        - Source of hcl nomad job
+      type: str
+'''
+
+EXAMPLES = '''
+- name: create job
```

Comment:
```suggestion
- name: Create job
```
etc. for all the examples